generators/server/templates/src/main/java/package/config/ReactiveSecurityConfiguration.java.ejs

```diff
@@ -266,6 +266,7 @@ public class SecurityConfiguration {
             .pathMatchers("/services/**", "/swagger-resources/**").authenticated()
             <%_ } _%>
             .pathMatchers("/management/health").permitAll()
+            .antMatchers("/management/health/**").permitAll()
```

Comment:
Are you sure it should not be `pathMatchers` here?

Reply:
Oh yes ! Good catch !

Reply:
What is even the difference ? Shouldn't we use antMatcher in all cases ?

Reply:
Oh it's for reactive. @pascalgrimmaud I think the reactive build is not enabled so this error was not caught !